Bio/PDB/__init__.py

```diff
@@ -88,3 +97,67 @@
     from .SASA import ShrakeRupley
 except ImportError:
     pass
+
+_FormatToParser = {"pdb": PDBParser(), "cif": MMCIFParser()}
+
+
+def read(handle, format, id=None):
+    """Parse a file or handle using one of the available parsers to generate a structure.
+
+    Arguments:
+     - handle    - handle to the file, or the filename as a string
+     - format    - string describing the file format. can be [pdb, cif, mmtf]
+     - id        - optional id of the structure, if None header information or then base name is used
```

Comment:
Do we need this capability? If a user for some reason doesn't want to use the id provided in the header information, it can always be changed after calling `PDB.read`.

Reply:
Just a quick note: this is necessary because not all files have an ID (field or otherwise).

Reply:
This was always a confusing point for me using the PDB parser - can the existing parsers not pick a sensible default or cope with a blank ID? It would be nicer not to ask the user here.

Reply:
The `id` field is just a friendly string to help you identify the structure you are working on. It has no "default" value or standard value from any file format (you could parse some sort of Id but it's inconsistent). I usually set it to the basename of the file I loaded.